### PR TITLE
Update registration questions

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
Should ask only for:
- Please enter your facility code
- Please select your gender: 
- Please enter your cadre
- Please enter your department name

No validation currently

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps please review

File: src/ussdapp.js [hunk 1]
```diff
@@ -19,15 +19,25 @@ go.app = function() {
...
+            "state_facility_code":
+                $("Please enter your facility code"),
+            "state_gender":
+                $("Please enter your gender:"),
+            "state_cadre":
+                $("Please enter your cadre"),
+            "state_department":
+                $("Please enter your department name"),
+            "state_end_registration":
+                $("Thank you for registering.  You'll soon be receiving quizzes."),
```
Comment (imsickofmaps) [inline src/ussdapp.js]: Double space

File: src/ussdapp.js [hunk 2]
```diff
@@ -40,75 +50,127 @@ go.app = function() {
...
         });
 
     // REGISTRATION STATES
 
         // FreeText st-01
-        self.add('state_id', function(name) {
+        self.add("state_facility_code", function(name) {
             return new FreeText(name, {
                 question: questions[name],
                 next: function(input) {
```
Comment (imsickofmaps) [inline src/ussdapp.js]: If we're not doing anything in the `next` functions, we should just simplify and set the next state as the value

**Comment by imsickofmaps** [comment]
Couple of tweaks then :+1: when travis is happy

**Pull request merged**